libkbfs/backpressure_disk_limiter.go

```diff
@@ -470,6 +479,49 @@ func (bdl *backpressureDiskLimiter) onBlocksDelete(
 	bdl.fileTracker.onBlocksDelete(blockFiles)
 }
 
+func (bdl *backpressureDiskLimiter) onDiskBlockCacheDelete(
+	ctx context.Context, blockBytes int64) {
+	if blockBytes == 0 {
+		return
+	}
+	bdl.lock.Lock()
+	defer bdl.lock.Unlock()
+	bdl.byteTracker.onBlocksDelete(blockBytes)
+}
+
+func (bdl *backpressureDiskLimiter) beforeDiskBlockCachePut(
+	ctx context.Context, blockBytes, diskBlockCacheBytes int64) (
```

Comment:
I think instead of passing in the total disk block cache size here, we need an `onDIskBlockCacheEnable` function. Otherwise, the disk limiter doesn't know about the bytes consumed by the disk cache except in this function.

That is, the bytes `backpressureTracker` should be keeping track of both the disk block cache and the journal bytes, i.e. the U in `U <= min(k(U+F), L)` in the comment for `backpressureTracker` would also count the disk block cache bytes.

Also, can you change the comment "Let U be the (approximate) resource usage of the journal..." to add "(and maybe the disk block cache)" near the top of the file?